wandb/sdk/internal/job_builder.py

```diff
@@ -121,8 +122,11 @@ def _build_repo_job(
                 "commit": commit,
             },
         }
-
-        name = make_artifact_name_safe(f"job-{remote}_{program_relpath}")
+        max_remote_name_length = (
+            MAX_ARTIFACT_NAME_LENGTH - len("job-") - len("_" + program_relpath)
```

Comment:
what happens if `program_relpath` is longer than `MAX_ARTIFACT_NAME_LENGTH`?

Reply:
switched to using regex